src/tryceratops/files/discovery.py

```diff
@@ -10,7 +10,7 @@
 
 from .parser import parse_file
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("tryceratops")
```

Comment:
I have a theory (that I need to test), that `__name__ ` would work the same way. I'll test later.